src/timeCalculator.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+
+function TimeCalculator(props) {
+  const [readingSpeed, setReadingSpeed] = useState(0);
+  const [pageCount, setPageCount] = useState(0);
+  const [wordsPerPage, setWordsPerPage] = useState(250);
+  const [time, setTime] = useState(0);
+  function handleChange(e) {
+    if (e.target.name === 'readingSpeed') {
```

Comment:
not sure if it'll work with hooks, but you can assign e.target.name and e.target.value to a variable to avoid using multiple conditionals to set state.

`const name = e.target.name, value = e.target.value
name(value)`